web/client/components/security/modals/LoginModal.jsx

```diff
@@ -54,17 +62,39 @@ class LoginModal extends React.Component {
     };
 
     getForm = () => {
-        return (<LoginForm
-            role="body"
-            ref="loginForm"
-            showSubmitButton={false}
-            user={this.props.user}
-            loginError={this.props.loginError}
-            onLoginSuccess={this.props.onLoginSuccess}
-            onSubmit={this.props.onSubmit}
-            onError={this.props.onError}
-        />);
-    };
+        const providers = this.props.providers.filter(({type}) => type === "basic");
```

Comment:
can we have more than one basic provider?
if so, how many forms do we need to show?

the logic here shouldn't be aligned as the one in the actions?
if we have (any) providers > 1 --> show login form

Reply:
especially because you just return one login form.

this confuses me a bit.

Reply:
No, at the moment only one basic (geostore) is allowed. 
The logic of the overall application is: 
If only one provider is present, and it is of type "openId", go directly on that login page.
Otherwise show the window (that may list the openId entries and/or the login form).